lib/resources/majors.tsx

```diff
@@ -2395,135 +2395,199 @@ const bsBME: Major = {
+      double_count: ['Writing Intensive', 'Major Requirement'],
+    },
+    {
+      name: 'Natural Science',
+      required_credits: 7,
+      min_credits_per_course: 1,
+      double_count: ['Writing Intensive', 'Major Requirement'],
+      description:
+        'Students must take two biology lecture courses and a biology lab course (Note: Other biology courses may apply with advisor approval)',
+      criteria:
+        'AS.020.151[C]^OR^AS.020.152[C]^OR^AS.020.303[C]^OR^AS.020.305[C]^OR^AS.020.306[C]^OR^AS.020.374[C]^OR^AS.280.161[C]^OR^' +
+        'AS.020.153[C]^OR^AS.020.154[C]^OR^AS.020.315[C]^OR^AS.020.316[C]^OR^AS.020.340[C]^OR^AS.020.377[C]^OR^AS.250.253[C]^OR^AS.250.254[C]',
+      fine_requirements: [
+        {
+          description:
+            '<b>Biology Lecture</b> <br /> AS.020.151 General Biology I <br /> <i>OR</i> <br /> AS.020.152 General Biology II <br /> <i>OR</i> <br />' +
+            'AS.020.303 Genetics <br /> <i>OR</i> <br /> AS.020.305 Biochemistry <br /> <i>OR</i> <br /> AS.020.306 Cell Biology <br /> <i>OR</i> <br />' +
+            'AS.020.374 Comparative Animal Physiology <br /> <i>OR</i> <br /> AS.280.161 Applications of Biological Concepts in Public Health',
+          required_credits: 6,
+          criteria:
+            'AS.020.151[C]^OR^AS.020.152[C]^OR^AS.020.303[C]^OR^AS.020.305[C]^OR^AS.020.306[C]^OR^AS.020.374[C]^OR^AS.280.161[C]',
+        },
+        {
+          description:
+            '<b>Biology Lab</b> <br /> AS.020.153 General Biology Laboratory I <br /> <i>OR</i> <br /> AS.020.154 General Biology Lab II' +
+            '<br /> <i>OR</i> <br /> AS.020.315 Biochemistry Project labs <br /> <i>OR</i> <br /> AS.020.316 Cell Biology Lab <br />' +
+            '<i>OR</i> <br /> AS.020.340 Developmental Genetics Lab <br /> <i>OR</i> <br /> AS.020.377 Comparative Physiology Lab <br />' +
+            '<i>OR</i> <br /> AS.250.253 Protein Engineering and Biochemistry Lab <br /> <i>OR</i> <br /> AS.250.254 Protein Biochemistry and Engineering Laboratory',
+          required_credits: 1,
+          criteria:
+            'AS.020.153[C]^OR^AS.020.154[C]^OR^AS.020.315[C]^OR^AS.020.316[C]^OR^AS.020.340[C]^OR^AS.020.377[C]' +
+            '^OR^AS.250.253[C]^OR^AS.250.254[C]',
+        },
+      ],
+    },
+    {
+      name: 'Quantitative',
+      required_credits: 4,
+      min_credits_per_course: 4,
+      double_count: ['All'],
+      description: 'Must complete Calculus I',
+      criteria: 'AS.110.106[C]^OR^AS.110.108[C]',
+      fine_requirements: [
+        {
+          description:
+            '<b>Calculus I</b> <br /> AS.110.106 Calculus I (Biology and Social Sciences) <br /> <i>OR</i> <br /> AS.110.108 Calculus I (Physical Sciences & Engineering)',
+          required_credits: 4,
+          criteria: 'AS.110.106[C]^OR^AS.110.108[C]',
+        },
+      ],
+    },
+    {
+      name: 'Writing Intensive',
+      required_credits: 15,
```

Comment:
could you please update all KSAS courses to be reintro to writing + 12 credits = required: 15?